orderer/common/server/main.go

```diff
@@ -442,6 +443,24 @@ func initializeClusterClientConfig(conf *localconfig.TopLevel) (comm.ClientConfi
 		UseTLS:            true,
 	}
 
+	if conf.General.Backoff.BaseDelay > 0 ||
```

Comment:
It seems we do not need to check "conf.General.Backoff.BaseDelay > 0 || conf.General.Backoff.Multiplier > 0 || conf.General.Backoff.MaxDelay > 0", because we check each item's value later.

Reply:
It's a test, made only for another.
This is a trigger that shows "are we going to change the default mechanism at all or not"